plugins/base/src/main/kotlin/translators/descriptors/DefaultDescriptorToDocumentableTranslator.kt

```diff
@@ -517,6 +532,25 @@ private class DokkaDescriptorVisitor( // TODO: close this class and make it priv
         )
     }.let(::Annotations)
 
+    fun DeclarationDescriptor.convertType(dri: DRI, context: DokkaContext): Bound = when (val desc = this) {
+        is ValueParameterDescriptor -> desc.type
+        is FunctionDescriptor -> desc.returnType
+        is PropertyDescriptor -> desc.returnType
+        is ReceiverParameterDescriptor -> desc.type
+        else -> null
+    }?.let { context.unresolvedTypeHandler(it) }.let { OtherParameter(it ?: "") }
+
+    inline fun <reified T: DeclarationDescriptor> Collection<T>.filterTypeError() =
+        if (context.unresolvedTypeHandler == UnresolvedTypeHandler.Skip) {
+            this.filterNot { it.isErrorType() }
+        } else this
+
+    fun DeclarationDescriptor.isErrorType() =
+        (this is FunctionDescriptor && this.returnType?.isError != false) ||
+                (this is ValueParameterDescriptor && this.type.isError) ||
+                (this is PropertyDescriptor && this.type.isError) ||
```

Comment:
This is a duplicated comment, but I feel that one of this places can be incorrect.